.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,90 @@
+name: Gemini Integrations Tests
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name:
+        uses: actions/checkout@v4
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: "1.23"
+      - name: Build
+        run: make build
+      - uses: actions/upload-artifact@v4
+        with:
+          name: bin
+          path: "./bin/gemini"
+          if-no-files-found: error
+          retention-days: 1
+  test:
+    needs: [build]
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        gemini-features: ["basic", "normal", "all"]
+        gemini-concurrency: [16]
+        oracle-scylla-version: ["5.2.19"]
+        test-scylla-version: ["5.2.19"]
```

Comment:
All these default scylla versions are too old. Need to make it be newer.
Also, is it intentional that oracle has the same version?

Reply:
> All these default scylla versions are too old. Need to make it be newer. Also, is it intentional that oracle has the same version?

It's just a placeholder for the review, wanted to sync will all to see what version to be used for testing and what for the oracle so it's just there to see it working